resources/testdata/glyphs2/WghtVar_ImplicitAxes.glyphs

```diff
@@ -20,8 +20,6 @@ capHeight = 0;
 descender = -200;
 id = m01;
 weightValue = 400;
```

Comment:
in fact, you'll notice how `weightValue = 400;` here is _not_ omitted by Glyphs.app (or glyphsLib) when serializing .glyphs source file, because != 100 (the actual omit-if-default value)